lib/engine/part/path.rb

```diff
@@ -152,6 +188,22 @@ def rotate(ticks)
         path
       end
 
+      def a_id
```

Comment:
i don't like the a_id methods and b_id methods, is it possible to call edge.id?

maybe change edge id so it works this?

Reply:
edge doesn't have access to lanes, which is why I made it part of Path.

Reply:
The problem is that lanes apply to things other than edges.

Reply:
I'm adding lanes to base and setting them in path.separate_parts